.github/actions/prepare-api-variables/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Prepare api variables
+description: Prepare env config for cloud
+
+inputs:
+  MODE:
+    description: "The mode of running tests (pull-request, release, main)"
+    required: true
+
+outputs:
+  BASE_URL:
+    description: "Dashboard base url"
+    value: ${{ steps.generate.outputs.BASE_URL }}
+  API_URL:
+    description: "API url"
+    value: ${{ steps.generate.outputs.API_URL }}
+  POOL_NAME:
+    description: "The name of the instance"
+    value: ${{ steps.generate.outputs.POOL_NAME }}
+  POOL_INSTANCE:
+    description: "The full URL of the instance"
+    value: ${{ steps.generate.outputs.POOL_INSTANCE }}
+  BACKUP_NAMESPACE:
+    description: "The backup namespace"
+    value: ${{ steps.generate.outputs.BACKUP_NAMESPACE }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Saleor login
+      uses: ./.github/actions/cli-login
+      with:
+        token: ${{ inputs.CLI_TOKEN }}
```

Comment:
Is this step actually needed? That composite action never talks to the Saleor CLI